### DISCUSSION TITLE
PACK PRO STUDENT

### DISCUSSION THREAD

**Discussion by thaovan535353** [community/community#45924] [GitHub Education]
### Select Topic Area

Question

### Body

Hello,
I'd like to register pack pro student Github for account's name: thaovan535353 (EMAIL REDACTED DUE TO PII). I coudn't complete to register this pack. You require me to o send additional proof of academic status. But I don't know where I can send it. Thank you for your instruction.

**Comment by nwuzmedoutlook** [comment]
I have the same problem.I do not know which email account to send additional proof of academic status to.🙄

**Comment by KuyaMecky** [comment]
i have the same problem
